docs/installation.md

```diff
@@ -0,0 +1,125 @@
+## Installation
```

Comment:
Seems legit. What if I renamed installation "customization"? Right now there's only custom plans but there will be also custom service defaults soon.